src/Components/Patient/FileUpload.tsx

```diff
@@ -301,6 +319,73 @@ export const FileUpload = (props: FileUploadProps) => {
     return ext;
   };
 
+  const getIconClassName = (extensionName: string | undefined) => {
+    // check for image files
+    if (
+      extensionName === ".png" ||
+      extensionName === ".jpg" ||
+      extensionName === ".jpeg" ||
+      extensionName === ".tif" ||
+      extensionName === ".tiff" ||
+      extensionName === ".bmp" ||
+      extensionName === ".eps" ||
+      extensionName === ".apng" ||
+      extensionName === ".avif" ||
+      extensionName === ".jfif" ||
+      extensionName === ".pjpeg" ||
+      extensionName === ".pjp" ||
+      extensionName === ".svg" ||
+      extensionName === ".webp"
```

Comment:
```suggestion
      [".png", ".jpg", ".jpeg", ".tif", ".tiff", ".bmp", ".eps", ".apng", ".avif", ".jfif", ".pjpeg", ".pjp", ".svg", ".webp"].some((ext) => ext === extensionName)
```

This should make it simple and easy to edit later as well